src/metadata.rs

```diff
@@ -41,18 +45,21 @@ where
     JA: JweKeyManagementAlgorithm + Clone,
 {
     credential_issuer: IssuerUrl,
-    authorization_server: Option<IssuerUrl>, // Not sure this is the right type
+    authorization_servers: Option<Vec<IssuerUrl>>, // Not sure this is the right type
```

Comment:
This appears to be the correct type as far as I can tell. You might drop the Option from here, as an empty Vec mostly implies the same thing, but its not 100% necessary.

Reply:
I think it's better to keep the `Option`, the semantic is the same, but the serialization is different with two different outputs:
```
{
  "credential_issuer": "https://some.url/issuer",
  "authorization_server": [],
}
```
vs
```
{
  "credential_issuer": "https://some.url/issuer"
}
```